common/esxi_get_host_facts.yml

```diff
@@ -3,10 +3,9 @@
 ---
 - name: Gather ESXi host facts
```

Comment:
We need to modify the task files in changing by the way to meet our convention and ansible-lint policies. e.g., name part with quotes "".